### PR TITLE
VZ-7424 Document the fact that CLI deploys the VPO

### PR DESCRIPTION
**Pull request by saketmht**

### CODE REVIEW

File: content/en/docs/setup/cli/_index.md [hunk 1]
```diff
@@ -63,16 +63,16 @@ vz [command] [flags]
...
-| `status`    | Status of the Verrazzano installation and access endpoints |
-| `uninstall` | Uninstall Verrazzano                                       |
-| `upgrade`   | Upgrade Verrazzano                                         |
-| `version`   | Verrazzano version information                             |
+| Command     | Definition                                                  |
+|-------------|-------------------------------------------------------------|
+| `analyze`   | Analyze cluster                                             |
+| `bug-report`| Collect information from the cluster to report an issue     |
+| `help`      | Help about any command                                      |
+| `install`   | Deploy Verrazzano Platform Operator and Install Verrazzano  |
```
Comment (rosemarymarano) [inline content/en/docs/setup/cli/_index.md:71]: re-write: Deploy the Verrazzano platform operator and install Verrazzano

File: content/en/docs/setup/cli/_index.md [hunk 2]
```diff
@@ -63,16 +63,16 @@ vz [command] [flags]
...
+| `status`    | Status of the Verrazzano installation and access endpoints  |
+| `uninstall` | Uninstall Verrazzano                                        |
+| `upgrade`   | Upgrade Verrazzano Platform Operator and Upgrade Verrazzano |
```
Comment (rosemarymarano) [inline content/en/docs/setup/cli/_index.md:74]: re-write: Upgrade the Verrazzano platform operator and upgrade Verrazzano

File: content/en/docs/setup/install/installation.md [hunk 3]
```diff
@@ -34,6 +34,9 @@ Verrazzano provides a platform [operator](https://kubernetes.io/docs/concepts/ex
 to manage the life cycle of Verrazzano installations.  Using the [Verrazzano]({{< relref "/docs/reference/api/verrazzano/v1beta1.md" >}})
 custom resource, you can install, uninstall, and upgrade Verrazzano installations.
 
+The Verrazzano CLI automatically deploys and installs the Verrazzano Platform Operator when applying the Verrazzano custom resource.
```
Comment (rosemarymarano) [inline content/en/docs/setup/install/installation.md:37]: re-write and add these sentences to the ones above it: When applying the Verrazzano custom resource, the Verrazzano CLI deploys and installs the Verrazzano platform operator; you need only to install Verrazzano as described in the following section.

**Review by rosemarymarano — COMMENTED** [review]
Please let me know when you have finished incorporating my review comments.

File: content/en/docs/setup/install/installation.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ You can install Verrazzano using the [Verrazzano CLI]({{< relref "docs/setup/cli
 
 Verrazzano provides a platform [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
 to manage the life cycle of Verrazzano installations.  Using the [Verrazzano]({{< relref "/docs/reference/api/verrazzano/v1beta1.md" >}})
-custom resource, you can install, uninstall, and upgrade Verrazzano installations.
+custom resource, you can install, uninstall, and upgrade Verrazzano installations. When applying the Verrazzano custom resource, the Verrazzano CLI deploys and installs the Verrazzano platform operator; you need only to install Verrazzano as described in the following section.
```
Comment (bobdonat) [inline content/en/docs/setup/install/installation.md:35]: This should be removed "When applying the Verrazzano custom resource".

File: content/en/docs/setup/cli/_index.md [hunk 5]
```diff
@@ -63,16 +63,16 @@ vz [command] [flags]
...
-| `status`    | Status of the Verrazzano installation and access endpoints |
-| `uninstall` | Uninstall Verrazzano                                       |
-| `upgrade`   | Upgrade Verrazzano                                         |
-| `version`   | Verrazzano version information                             |
+| Command     | Definition                                                      |
+|-------------|-----------------------------------------------------------------|
+| `analyze`   | Analyze cluster                                                 |
+| `bug-report`| Collect information from the cluster to report an issue         |
+| `help`      | Help about any command                                          |
+| `install`   | Deploy the Verrazzano platform operator and install Verrazzano  |
```
Comment (bobdonat) [inline content/en/docs/setup/cli/_index.md:71]: The vz CLI help shows the same thing for install and upgrade.  If we think this needs changing then probably the vz CLI help should also be changed.

Comment (saketmht) [reply content/en/docs/setup/cli/_index.md:71]: Will create a jira for that

**Pull request merged**